pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -54,14 +55,15 @@ func (r *ConnectionRecord) isValid() error {
 }
 
 // NewConnectionRecorder returns new connection record instance
-func NewConnectionRecorder(transientStore, store storage.Store) *ConnectionRecorder {
-	return &ConnectionRecorder{transientStore: transientStore, store: store}
+func NewConnectionRecorder(transientStore, store storage.Store, didMap didconnection.Store) *ConnectionRecorder {
+	return &ConnectionRecorder{transientStore: transientStore, store: store, didMap: didMap}
 }
 
 // ConnectionRecorder takes care of connection related persistence features
 type ConnectionRecorder struct {
 	transientStore storage.Store
 	store          storage.Store
+	didMap         didconnection.Store
```

Comment:
why didMap rather than didStore?

Reply:
This is an alternate name I had for it. Consistency suggests that they should match, but I'm still unsure what name to use. DIDMap? didconnection.Store? didlookup.Store? All alternatives I've tried using already.

Reply:
Renamed to didStore.